app/models/student.rb

```diff
@@ -0,0 +1,13 @@
+class Student < ApplicationRecord
+  belongs_to :course
+  belongs_to :teacher
+  validates :first_name, :last_name, :birthday, :year, :address, presence: true
+
+  # to be edited
+  enum emergency_contact:[
```

Comment:
maybe make phone number and emergency contact name an attribute under student instead of enum?